build.gradle.kts

```diff
@@ -31,6 +31,21 @@ import de.thetaphi.forbiddenapis.gradle.CheckForbiddenApis
 import de.thetaphi.forbiddenapis.gradle.CheckForbiddenApisExtension
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 
+buildscript {
+
+    dependencies {
+        constraints {
+            // FIXME Should use asm.version from properties
+            "classpath"("org.ow2.asm:asm::9.7.1")
```

Comment:
Not too familiar with gradle, but is there anyway to avoid hardcoding the versions in here?

Reply:
Neither I am.
I have tried a few things, but I was not able to get anything working.

Reply:
So if there is one, I couldn't find it.

Reply:
I suspect that as this is used for initializing gradle, you cannot even access properties or run kotlin code in this block.

Maybe I should remove the FIXME comment, as using properties here may not even be possible.